journey/k8s.ts

```diff
@@ -1,52 +1,50 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-FileCopyrightText: 2023-Present The Pepr Authors
 
-import { AppsV1Api, CoreV1Api, KubeConfig, V1ConfigMap } from "@kubernetes/client-node";
+import { K8s, kind } from "kubernetes-fluent-client";
 
-const kc = new KubeConfig();
-kc.loadFromDefault();
+import { PeprStore } from "../src/lib/k8s";
 
-const k8sApi = kc.makeApiClient(AppsV1Api);
-const k8sCoreApi = kc.makeApiClient(CoreV1Api);
-
-function delay2Secs() {
+export function delay2Secs() {
   return new Promise(resolve => setTimeout(resolve, 2000));
 }
```

Comment:
```suggestion
export function sleep(seconds: number) {
  return new Promise(resolve => setTimeout(resolve, seconds*1000));
}
```

idk, when I do stuff like this, I like calling the fn `sleep` so it looks just like Python's `time.Sleep(n)`